README.md

```diff
@@ -18,7 +18,7 @@ The current Swift Server work group consists of the following people:
 * Patrick Freed ([@patrickfreed](https://github.com/patrickfreed), MongoDB)
 * Simon Pilkington ([@tachyonics](https://github.com/tachyonics), Amazon)
 * Tim Condon ([@0xTim](https://github.com/0xTim), Vapor)
-* Todd Varland ([@toddvarland](https://github.com/toddvarland), Amazon)
+* Jimmy McDermott ([@jdmcd](https://github/com/jdmcd), Transeo)
```

Comment:
should this point to swift.org version of this file? I think there is some duplicate info and would be better to have one source of truth

Reply:
https://www.swift.org/sswg/

Reply:
Yeah I think that would be smart - opened a PR: https://github.com/swift-server/sswg/pull/79